### ISSUE TITLE
Dependencies in PyPi release

### ISSUE THREAD

**Issue by Dacit** [inpefess/isabelle-client#30]
For the 0.2.3 pypi release, the dependencies seem to be missing - at least the `dataclasses` package is required:

```python
>>> from isabelle_client.utils import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/virtualenvs/isadjango/lib/python3.6/site-packages/isabelle_client/__init__.py", line 17, in <module>
    from isabelle_client.isabelle__client import IsabelleClient
  File "/virtualenvs/isadjango/lib/python3.6/site-packages/isabelle_client/isabelle__client.py", line 22, in <module>
    from isabelle_client.socket_communication import (
  File "/virtualenvs/isadjango/lib/python3.6/site-packages/isabelle_client/socket_communication.py", line 18, in <module>
    from dataclasses import dataclass
ModuleNotFoundError: No module named 'dataclasses'
```

**Comment by inpefess** [comment]
Thank you for reporting that! That happens only for Python 3.6 since I install `dataclasses` as a development dependency of something else.
I released `0.2.4` version with a conditional dependency on `dataclasses` added. Try now please.

**Comment by Dacit** [comment]
Thanks, that solves it.
